Source/Hypermedia.Client.Extensions/NewtonsoftJson/NewtonsoftJson.csproj

```diff
@@ -33,8 +33,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Bluehands.Hypermedia.Client" Version="1.5.0" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
This seems a accident. Don't we keep the nuget package?

Reply:
For development not, it will have to be updated to the new version of the nuget package when it is released. But this way anyone checking out the branch can actually build the current state.